### DISCUSSION TITLE
Please keep the 32 bit support for Redot

### DISCUSSION THREAD

**Discussion by Dynacato** [Redot-Engine/redot-proposals#26] [Ideas]
Keep the original 32 bit support of every device where the editor is available.

In less developed countries, 64 bit is still not entirely adapted, limiting the engine usage in that cases.
Also the implementation will increase the compatibility for more devices when exporting projects to specific platforms.
Godot code currently supports it, the implementation will not be as difficult while Redot is still in an initial state as opposed to a stage where it derives more from the original repository.

Mak Gam, even in 32 bit

**Comment by Arctis-Fireblight** [comment]
No plans to drop 32 bit entirely, but we have seen very low adoption for 32bit builds. 
Starting with Redot LTS 26.1 32bit users will need to build from source, as we are no longer distributing pre-compiled binaries due to low adoption.
